modules/@apostrophecms/notification/ui/apos/components/TheAposNotifications.vue

```diff
@@ -92,37 +92,42 @@ export default {
     },
     async poll() {
       try {
-        const allNotifications = [ ...this.notifications, ...this.dismissed ];
-        const latestTimestamp = allNotifications
-          .map(notification => notification.updatedAt)
-          .sort()
-          .reverse()[0];
+        if (document.visibilityState === 'hidden') {
+          // Wait for tab to become visible
+          setTimeout(this.poll, 5000);
+        } else {
+          const allNotifications = [ ...this.notifications, ...this.dismissed ];
```

Comment:
git thinks I changed a lot here but all I really did was nest in a check for visibility.